lib/MusicBrainz/Server/Data/Artist.pm

```diff
@@ -228,9 +228,24 @@ sub _order_by {
         "name" => sub {
             return "musicbrainz_collate(name)"
         },
+        "area" => sub {
+            return "area, musicbrainz_collate(name)"
```

Comment:
Shouldn't we join the area table and sort by its name instead of its ID? 'Cause I imagine the ID won't have any logical order other than putting identical areas next to each other.

Reply:
Maybe? This is what we have done all the time, but it doesn't seem too smart. Of course the ideal sort would use containment but name might be better than id anyway

Reply:
I guess just labels, but I see we do similar bad stuff with gender, type, etc. below. Surprised no one's complained about that. Anyway, if you're okay with keeping it how it is for now we can merge this and I'll enter a ticket for having the sort make sense.